EarTrumpet/DataModel/WindowsAudio/Internal/AudioDeviceSession.cs

```diff
@@ -189,6 +192,9 @@ public AudioDeviceSession(IAudioDevice parent, IAudioSessionControl session, Dis
                 SyncPersistedEndpoint(parent);
             }
         }
+
+        // Potential memory leak: this class is not IDisposable, so we cannot unregister the event.
+        App.Settings.UseLogarithmicVolumeChanged += (sender, args) => RaisePropertyChanged(nameof(Volume));
```

Comment:
No real leak, App.Settings holds a single static instance throughout the lifetime of the app. It, and all registered event handlers, will get cleaned up forcefully at application exit.